tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -475,6 +475,24 @@ def wait_for_replay_complete(self, imagespec):
                 if int(out.split("=")[-1]) == 0:
                     return 0
 
+    def wait_for_snapshot_complete(self, imagespec):
+        """verify for snapshot completion
+        Args:
+            imagespec: Image specification.
+        """
+        while True:
```

Comment:
Could you please add an optimistic timeout, maybe some 300 to 600 seconds, post which if snapshot isn't complete, it exits from the method. Else this loop will continue indefinitely right?

Reply:
Added timeout as 600 sec.

Reply:
LGTM